apps/laboratory/tests/shared/fixtures/w3m-fixture.ts

```diff
@@ -12,6 +14,10 @@ export const testM = base.extend<ModalFixture>({
     const modalPage = new ModalPage(page)
     await modalPage.load()
     await use(modalPage)
+  },
+  modalValidator: async ({ modalPage }, use) => {
+    const modalValidator = new ModalValidator(modalPage.page)
+    await use(modalValidator)
   }
 })
 export { expect } from '@playwright/test'
```

Comment:
Any reason to export `expect` here vs importing it where it is needed, is this part of playwright's api or something similar?

Reply:
I copied the code from `V2`. I will assess your feedback and refactor in the next iteration. Multiple patterns here that I think can be improved.

I expect this test suite to become very large so important that we have a good setup